### PR TITLE
chore: brush storybook

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* add brush story

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/brushes/axis.ts [hunk 1]
```diff
@@ -1,38 +1,15 @@
...
-
-export function line(ctx: CanvasRenderingContext2D, lineModel: LineModel) {
-  const { x, y, x2, y2 } = lineModel;
-
-  ctx.beginPath();
-  ctx.moveTo(x, y);
-  ctx.lineTo(x2, y2);
-  ctx.stroke();
-  ctx.closePath();
+  line(ctx, lineModel);
```
Comment (shiren) [inline src/brushes/axis.ts:14]: good

File: src/brushes/tooltip.ts [hunk 2]
```diff
@@ -27,36 +31,56 @@ export function tooltip(ctx: CanvasRenderingContext2D, tooltipModel: TooltipMode
...
+      style: [
+        'default',
+        {
+          textBaseline: 'top',
+          fillStyle: '#fff',
+          font: 'bold 13px Arial',
+          textAlign: 'left'
+        }
+      ]
+    });
```
Comment (shiren) [inline src/brushes/tooltip.ts:48]: good

File: stories/brush.stories.ts [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+export const lineBrush = () => {
+  const { ctx, el } = setup();
+
+  const lineModel: LineModel = { type: 'line', x: 100, y: 100, x2: 200, y2: 200 };
+  line(ctx, lineModel);
+
+  return el;
+};
+
+export const tickBrush = () => {
```
Comment (jungeun-cho) [inline stories/brush.stories.ts:111]: title이 되는 변수명을 Brush를 제거해주는건어때요?
Brushes/Tick 이런식으로 노출되도록이요.

File: src/component/plot.ts [hunk 4]
```diff
@@ -15,14 +15,14 @@ export default class Plot extends Component {
...
-  renderModels(relativePositions: number[], vertical: boolean): PlotModel[] {
+  renderModels(relativePositions: number[], vertical: boolean): LineModel[] {
     return relativePositions.map(position => {
       const x = vertical ? crispPixel(position) : crispPixel(0);
       const y = vertical ? crispPixel(0) : crispPixel(position);
       const width = vertical ? 0 : this.rect.width;
       const height = vertical ? this.rect.height : 0;
 
-      return { type: 'plot', x, y, width, height };
+      return { type: 'line', x, y, x2: x + width, y2: y + height };
```
Comment (jungeun-cho) [inline src/component/plot.ts:25]: 👍  plot 브러시를 제거하고 line 브러시를 재사용했군요!

File: stories/brush.stories.ts (hunk 3 above)
Comment (jung-han) [reply stories/brush.stories.ts:111]: @jungeun-cho 
tick이 이미 브러시 함수 이름이여서 중복되게 작성을 못해서 위처럼 작성했는데.. 혹시 다른 아이디어가 있을까요.

**Pull request merged**